src/v/redpanda/admin/api-doc/transform.json

```diff
@@ -0,0 +1,155 @@
+{
+  "apiVersion": "0.0.1",
+  "swaggerVersion": "1.2",
+  "basePath": "/v1",
+  "resourcePath": "/transform",
+  "apis": [
+    {
+      "path": "/v1/transform/deploy",
+      "operations": [
+        {
+          "method": "POST",
+          "summary": "Deploy a transform based on the provided metadata and wasm binary in the request payload. The payload should be a json `deploy_transform_request` object immediately followed by the wasm binary without any delimiters.",
```

Comment:
nit: it is strange to me to put the request payload type in the summary since user's do not see the `deploy_transform_request` type.

Reply:
Ah okay removed.